.ci/docker/common/install_ucc.sh

```diff
@@ -60,6 +66,7 @@ function install_ucc() {
     for arch in $amdgpu_targets; do
       HIP_OFFLOAD="$HIP_OFFLOAD --offload-arch=$arch"
     done
+    HIP_OFFLOAD="$HIP_OFFLOAD --rocm-path=${ROCM_PATH} --rocm-device-lib-path=${ROCM_PATH}/lib/llvm/amdgcn/bitcode"
```

Comment:
Let's define `ROCM_DEVICE_LIB_PATH` in `detect_rocm_path.sh` and use it here, since the actual path depends on whether it's a legacy installation of ROCm or not